src/sentry/issues/ongoing.py

```diff
@@ -11,28 +13,31 @@
 from sentry.types.group import GroupSubStatus
 
 
-def transition_new_to_ongoing(group: Group) -> None:
-    if group.status == GroupStatus.UNRESOLVED and group.substatus == GroupSubStatus.NEW:
-        _transition_group_to_ongoing(group)
-
-
-def transition_regressed_to_ongoing(group: Group) -> None:
-    if group.status == GroupStatus.UNRESOLVED and group.substatus == GroupSubStatus.REGRESSED:
-        _transition_group_to_ongoing(group)
-
-
-def _transition_group_to_ongoing(group: Group) -> None:
-    group.substatus = GroupSubStatus.ONGOING
-    group.save(update_fields=["substatus"])
-
-    remove_group_from_inbox(group)
-
-    add_group_to_inbox(group, GroupInboxReason.ONGOING)
-
-    Activity.objects.create_group_activity(
-        group, ActivityType.AUTO_SET_ONGOING, send_notification=False
-    )
-
-    record_group_history_from_activity_type(
-        group, activity_type=ActivityType.AUTO_SET_ONGOING.value, actor=None
-    )
+def transition_group_to_ongoing(
+    from_status: GroupStatus, from_substatus: GroupSubStatus, group: Group
+) -> None:
+    # make sure we don't update the Group when its already updated by conditionally updating the Group
```

Comment:
I don't think I understand what you mean here

Reply:
This is mostly a comment about the conditional update: https://github.com/getsentry/sentry/pull/49522/files#diff-ac34507346b0ea296e2173a670dc216ee9dd1cb717dd8d4f31c6dff86d4a5beeR20-R22

Since the group can be updated from various code-paths, we wanna make sure the Group status and substatus values haven't changed since this method was called.